Language.md

```diff
@@ -1 +1,74 @@
-# COMING SOON
+# JavaScript &#10084;
+A famous man once said JavaScript is the Language made for the Gods.
+Okay I said  it and I am neither famous nor very smart, but going by the stats JavaScript is *the* most famous language in the world right now.
+So they must be doing something right.
+
+So getting right down to the facts. Some of the things I find really useful about JavaScript:
+
+**1. It's made for the web**
+
+More like the web is made of JavaScript.
+**Literally**.Almost every website you come across has JavaScript in some way or form. In the early days when the Java guys were trying to get their own version of the web started (*applets*,ahem ahem).
+
+A relatively unknown language called EcmaScript was born. It had it's quirks and bugs for sure. Poor design decisions? Absolutely .But it took a hold of the programmers like no other.
+The brilliant thing about JavaScript was it was designed from the ground up with the internet in mind ,which sadly Java did not as they tried to shoehorn this extra 'web functionality' into their admittedly admirable language (I like all things Java, as you might have guessed)
+
+**2. It's evolving like crazy**
+
+JavaScript is changing at a rapid pace, what was cool six months ago has been replaced by a leaner and meaner counterpart. *Gulp* replaces *Grunt*.*Socket.io* replaces *Web-RTC* and so on and so forth.
```

Comment:
Socket.io and WebRTC aren't really entirely comparable. A better example would be 

Promises replace Callbacks. Async Await replace Promises...  or something like that.